packages/lesswrong/lib/analyticsEvents.tsx

```diff
@@ -328,6 +329,10 @@ function browserConsoleLogAnalyticsEvent(event: any, rateLimitExceeded: boolean)
   for (let fieldName of Object.keys(event.props)) {
     c.log(`${fieldName}:`, event.props[fieldName]);
   }
+  // Timestamp recorded on the server will differ. Obviously in part because of
+  // the latency of the network, but also because we have a queue that we only
+  // flush max once/second.
```

Comment:
The main reason for mismatched timestamps is actually clock skew (or, in the case of a development machine, imprecise correction for clock skew). When the events are sent to the server, each of them has a time stamp, and then the submission as a whole has a timestamp for what time the client thinks it is. The server adjusts the time on each of the events to reflect the skew implied by that timestamp.

Reply:
Wait really? My clock doesn't seem to be more than half a seconds skew. Maybe I wouldn't notice. It seems bang on with my phone, say.